src/pages/getting-started/app-guides/onboarding-and-settings/index.mdx

```diff
@@ -0,0 +1,73 @@
+desc: "Use the full potential of LiveChat onboarding features to increase sales and retention of your app. Add onboarding instructions, remind users to complete app setup, prepare a tutorial, and add app settings."
+---
+
+Onboarding & Settings is a building block helping you with the successful onboarding of your app users. Also, this block allows you to define app settings, where the users will be able to configure your application.
+
+# Building block configuration
+
+## Onboarding
+
+Sometimes, despite good UX patterns in the app, the initial setup is a bit more complicated and, as a result, some users get lost, frustrated, or even drop out and uninstall the app. To prevent that from happening, you can use three ways to onboard your app users:
+
+- [Setup completion endpoint](#setup-completion-endpoint)
+- [Onboarding guidelines](#onboarding-guidelines)
+- [Tutorial URL](#tutorial-url)
+
+### Setup completion endpoint
+
+Setup completion endpoint is unique for your application. Call the endpoint it to inform us that a user hasn’t completed the app setup after the installation (for example, didn’t connect with a third-party service). Thanks to this info, **we’ll be able to remind the user to finish the configuration next time they visit your app page on the Marketplace.**
+
+You can use HTTP PUT and DELETE methods. Use PUT to add a user whom you want to remind about app configuration, and make a DELETE call when a user has completed the setup and no longer needs the reminder.
+
+<CodeSample path={'Sample PUT request to the /setup endpoint'}>
+
+```bash
+curl -X PUT 'https://dev-platform.livechatinc.com/v2/applications/15jS39tMg/instance/setup' \
+-H 'Authorization: Bearer <access_token>'
+```
+
+</CodeSample>
+
+ This functionality requires the [App Authorization block](/getting-started/app-guides/app-authorization). You should call `/setup` with the `access token` of the user who gave their permission to your app.
+
+<div style={{maxWidth: "600px"}}>
+  <img alt="Onboarding " src="/images/getting-started/app-guides/onboarding-and-settings/complete-setup-prompt-livechat.png" width="1416px" height="1048px"/>
+</div>
+
+### Onboarding guidelines
+
+Another way of educating your app users is including onboarding guidelines. You can write them in markdown directly in Developer Console. We'll show them on your app page on the Marketplace to those users who installed your app and in your app settings in the LiveChat application.
+
+Go thorugh the following list and see what kind of inromation you might ant to include in your guidelines:
+
+- What to do/where to go after the app installation.
+- Any required registration with a third-party service.
+- Providing user credentials, such as API keys, phone numbers, etc.
+
+<div style={{maxWidth: "600px"}}>
+  <img alt="Onboarding " src="/images/getting-started/app-guides/onboarding-and-settings/onboarding-guidelines-duo3.png" width="1670px" height="1772px"/>
```

Comment:
I was trying hard to show "before and after", but let's be honest, this image duo doesn't look too good